src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx

```diff
@@ -55,6 +55,7 @@ export const ColorPickerColorsGrid = React.forwardRef(
               colorSize={colorSize}
               tooltipContent={tooltipContentByColor[color]}
               colorShape={colorShape}
+              data-testid={`color-picker-item-${color}`}
```

Comment:
I saw we create get test id function, should we use it? it create a data test id which depend on element type enum and also the id